fastlane_bot/bot.py

```diff
@@ -803,15 +808,16 @@ def calculate_profit(
             self.ConfigObj.logger.debug(f"[bot.calculate_profit sorted_price_curves] {sorted_price_curves}")
             if len(sorted_price_curves)>0:
                 fltkn_eth_conversion_rate = sorted_price_curves[0][-1]
-                best_profit_eth = Decimal(str(best_profit_fl_token)) / Decimal(str(fltkn_eth_conversion_rate))
+                flashloan_fee_amt_eth = Decimal(str(flashloan_fee_amt_fl_token)) / Decimal(str(fltkn_eth_conversion_rate))
+                best_profit_eth = Decimal(str(best_profit_fl_token)) / Decimal(str(fltkn_eth_conversion_rate)) - flashloan_fee_amt_eth
```

Comment:
Possibly better to avoid dividing by the ETH conversion rate here, and leave it to take place (if really necessary) at a different point during execution?

If yes, then the two lines above become somewhat simpler, cleaner, and most importantly - more accurate:
```
flashloan_fee_amt_eth = flashloan_fee_amt_fl_token
best_profit_eth = best_profit_fl_token - flashloan_fee_amt_fl_token
```

Reply:
I am pretty sure I already responded to this comment somewhere but dont see it now... In any case, because the profit is calculated and returned within this function, I dont know why we would (or should) treat the flashloan fee any different than the rest of the profit calculation(s)?

Once the calculated profit leaves this function it is compared against the gas costs and the user-specified minimum profit (in simple great-than, less-than comparisons). IMO the most intuitive thing to do is keep the profit calculation contained within a single function like it is now, and not delay part of the calculation to happen elsewhere.